transformer_engine/jax/transformer.py

```diff
@@ -185,53 +185,55 @@ class MultiHeadAttention(nn.Module):
     Parameters
     ----------
     head_dim : int
-        the hidden dimension of each attention heads.
+        The hidden dimension of each attention head.
     num_heads : int
-        the number of attention heads
+        The number of attention heads
     dropout_rate : float, default = 0.0
-        dropout probability for the dropout op during multi-head attention.
+        Dropout probability for the dropout op during multi-head attention.
     dropout_rng_name: str, default = 'dropout'
-        the key in given RNGs via flax.linen.Module.apply that
-        for generate Dropout masks in the core attention.
+        The key in given RNGs via flax.linen.Module.apply that
+        to generate Dropout masks in the core attention.
```

Comment:
```suggestion
        The key in given RNGs via flax.linen.Module.apply that are
        used to generate Dropout masks in the core attention.
```

Reply:
Is this what it means? @mingxu1067

Reply:
Yes, it is. Change to the suggestion. THX

Reply:
Fixed